### PR TITLE
User Guide

### PR DESCRIPTION
**Pull request by dmurvihill**
Deal with Lauer's comments on the user guide

### CODE REVIEW

File: doc/src/user_guide.ltx [hunk 1]
```diff
@@ -38,16 +38,16 @@ following URL:
 \url{https://www.java.com/en/download/index.jsp}
 \end{figure}
 
-Please install the 64-bit version of the JVM if your system supports it.
-\textit{Checksims} can consume a substantial amount of memory, potentially more
-than the 4GB maximum that a 32-bit JVM can use. A 64-bit JVM can prevent a
-number of memory-related program crashes.
+The 64-bit version of the JVM is strongly recommended.
```
Comment (mheon) [inline doc/src/user_guide.ltx]: Recommend prefacing with "If your system supports it," for clarity

Comment (dmurvihill) [inline doc/src/user_guide.ltx]: What are we trying to clarify? If we really want to _strongly_ recommend a 64-bit JVM, don't you think we should do that regardless of what hardware the user has?

I wouldn't want to try running S-W on a Pentium 4...

Comment (mheon) [inline doc/src/user_guide.ltx]: Perhaps say that we strongly recommend a 64-bit system and 64-bit JVM?

**Comment by dmurvihill** [comment]
Okay.

File: doc/src/user_guide.ltx (hunk 1 above)
Comment (mheon) [inline doc/src/user_guide.ltx]: Hell, we should type up min system requirements at some point, but later

**Comment by dmurvihill** [comment]
Merge please?

**Pull request merged**